extensions/README.md

```diff
@@ -39,19 +39,21 @@ If your block depends on another block, place them all in extensions folder:
 1. Use the [Jetpack Docker environment](https://github.com/Automattic/jetpack/tree/master/docker#readme).
 1. Start a new branch.
 1. Add your new extension's source files to the extensions/blocks directory.
-And add your extensions's slug the appropriate array in `extensions/index.json`.
+And add your extensions's slug the beta array in `extensions/index.json`.
+By keeping your extension in beta array, it's safe to do small PRs and merge frequently.
 1. Or modify existing extensions in the same folder.
 1. Run `yarn build-extensions [--watch]` to compile your changes.
-1. Now test your changes in your docker environment's wp-admin.
+1. Now test your changes in your Docker environment's wp-admin.
 1. Open a PR, and a WordPress.com diff will be automatically generated with your changes.
 1. Test the WordPress.com diff
 1. Once the code works well in both environments and has been approved by a Jetpack crew member, you can merge your branch!
+1. When your block is ready to be shipped, move your extensions's slug from beta to production array in `extensions/index.json`
 
 ### Beta Extensions
 Generally, all new extensions should start out as beta.
 
 - Before you develop, remember to add your extension's slug to the beta array in `extensions/index.json`.
```

Comment:
Should we add a mention of the WP Cli command at this point, or even higher on the page? We don't really want people to start following this step-by-step guide only to find out when they reach the last step that there was a whole paragraph about automating this downer on the page.

Reply:
Good point. Mentioned even earlier and moved the whole section up a bit https://github.com/Automattic/jetpack/pull/12050/commits/cf46a470d7e13743bcb6b052e464a359348fc6e1